README.md

```diff
@@ -17,7 +16,7 @@ style.
 
 ### Textmate 2.x
 
-    git clone https://github.com/pmeinhardt/tm-github-theme.git /Users/pmeinhardt/Library/Application\ Support/TextMate/Managed/Bundles/Themes.tmbundle/Support/web-themes/github
+    git clone https://github.com/pmeinhardt/tm-github-theme.git /Users/<your-username>/Library/Application\ Support/TextMate/Managed/Bundles/Themes.tmbundle/Support/web-themes/github
```

Comment:
I should've thought about this before. We could use `HOME` from the shell environment so it works for all users:

```
$HOME/Library/Application\ Support/TextMate/Managed/Bundles/Themes.tmbundle/Support/web-themes/github
```

Reply:
Yup, thats better.